client/pages/me/me.js

```diff
@@ -1,66 +1,85 @@
 // pages/me/me.js
-Page({
+const qcloud = require('wafer2-client-sdk');
+const app = getApp();
 
-  /**
-   * 页面的初始数据
-   */
+Page({
   data: {
-
+    personal: {},
+    chineseNameInputValue: null,
+    englishNameInputValue: null,
+    phoneInputValue: null,
+    emailInputValue: null,
+    btnDisable: true
   },
-
-  /**
-   * 生命周期函数--监听页面加载
-   */
-  onLoad: function (options) {
-
+  onLoad: function () {
+    this.setData({
+      personal: app.userInfo
+    });
   },
 
-  /**
-   * 生命周期函数--监听页面初次渲染完成
-   */
-  onReady: function () {
-
+  chineseNameInput: function (event) {
+    if (event.detail.value.length > 0) {
+      this.setData({
+        chineseNameInputValue: event.detail.value,
+        btnDisable: false
+      });
+    }
   },
-
-  /**
-   * 生命周期函数--监听页面显示
-   */
-  onShow: function () {
-
+  englishNameInput: function (event) {
+    if (event.detail.value.length > 0) {
```

Comment:
If the original value is `Max`, and I'm deleting the three characters one by one before I click save, the final value of `englishNameInputValue` will be 'M', but the input field is empty on the screen. This isn't what we want.
Instead of not changing the value, I think we can disable the button if length is 0:
```
this.setData({
  englishNameInputValue: event.detail.value,
  btnDisable: event.detail.value.length === 0
});
```